src/main/java/com/google/api/codegen/transformer/java/JavaGapicSurfaceTransformer.java

```diff
@@ -85,6 +87,8 @@
   private static final String PAGE_STREAMING_RESPONSE_TEMPLATE_FILENAME =
       "java/page_streaming_response.snip";
 
+  private static final String DEFAULT_VERSION = "UNKNOWN";
```

Comment:
@shinfan I guess I already knew about this PR. :)

Please note that I have comments on this and https://github.com/googleapis/gax-java/pull/203 about changing `"UNKNOWN"` to `""`, which is the decision we agreed upon this week with the service controller folks.